docker-compose.kibana.yaml

```diff
@@ -17,9 +17,10 @@ services:
     volumes:
       - ./kibana/config.yml:/usr/share/kibana/config/kibana.yml
     depends_on:
-      - elasticsearch
+      elasticsearch:
+        condition: service_healthy
     healthcheck:
-      test: [ "CMD-SHELL", "curl --fail -s localhost:5601" ]
+      test: [ "CMD-SHELL", "curl --fail -s http://localhost:5601/api/status || exit 1" ]
```

Comment:
What is 5601? And you shouldn't be able to access it from the host unless it's set in `ports`... and it shouldn't be set in `ports`. 

Use a standardized way to access the elasticsearch container if that's what is being attempted here. Maybe `curl elasticsearch:5601/api/status` ?

Reply:
the port 5601 is the default port for kibana, but i think in these scenerio we need more complex healthcheck.

not only check if the kibana container has a healthy status, but as you mentioned before also checking elasticsearch app itself not only container  status healthcheck, 

i think of removing this test and replacing it with a shell script to check the elastic search and kibana app if they have a green status